php/commands/user.php

```diff
@@ -661,11 +678,34 @@ public function list_caps( $args, $assoc_args ) {
 
 			$user_caps_list = $user->allcaps;
 
+			$active_user_cap_list = array();
+
 			foreach ( $user_caps_list as $cap => $active ) {
 				if ( $active ) {
-					\cli\line( $cap );
+					$active_user_cap_list[] = $cap;
+				}
+			}
+
+			if ( ! empty( $active_user_cap_list ) ) {
```

Comment:
I think we should always display the list, even when there are no caps. This is consistent with the other commands that implement `--format=<format>`.

By always displaying the list, you'd get an empty table with `--format=table`